lib/code_corps/github/installation.ex

```diff
@@ -61,7 +61,7 @@ defmodule CodeCorps.GitHub.Installation do
   def refresh_token(%GithubAppInstallation{github_id: installation_id} = installation) do
     endpoint = "installations/#{installation_id}/access_tokens"
     with {:ok, %{"token" => token, "expires_at" => expires_at}} <-
-           GitHub.authenticated_integration_request(%{}, :post, endpoint, %{}, []),
+           GitHub.integration_request(:post, endpoint, %{}, %{}, []),
```

Comment:
I went with the `method, url, headers, body, options` order of arguments for all request functions where it applies. My reasoning is that `:hackney` uses the same ordering.